docs/refs.bib

```diff
@@ -0,0 +1,51 @@
+Mitigations.rst
+
+Agarwal, Beygelzimer, Dudik, Langford, Wallach “A Reductions Approach to Fair Classification”, ICML, 2018.
+@article{agarwal2018reductions,
+  author    = {Alekh Agarwal and
+               Alina Beygelzimer and
+               Miroslav Dudík and
+               John Langford and
+               Hanna M. Wallach},
+  title     = {A Reductions Approach to Fair Classification},
+  journal   = {ICML},
+  volume    = {abs/1803.02453},
+  year      = {2018},
+  eprinttype = {arXiv},
+  eprint    = {1803.02453},
+  timestamp = {Mon, 13 Aug 2018 16:47:08 +0200},
+  biburl    = {https://dblp.org/rec/journals/corr/abs-1803-02453.bib},
+  bibsource = {dblp computer science bibliography, https://dblp.org}
+}
+
+Agarwal, Dudik, Wu “Fair Regression: Quantitative Definitions and Reduction-based Algorithms”, ICML, 2019.
+@article{agarwal2019fair,
+  author    = {Alekh Agarwal and
+               Miroslav Dudík and
+               Zhiwei Steven Wu},
+  title     = {Fair Regression: Quantitative Definitions and Reduction-based Algorithms},
+  journal   = {ICML},
+  volume    = {abs/1905.12843},
+  year      = {2019},
+  eprinttype = {arXiv},
+  eprint    = {1905.12843},
+  timestamp = {Mon, 03 Jun 2019 13:42:33 +0200},
+  biburl    = {https://dblp.org/rec/journals/corr/abs-1905-12843.bib},
+  bibsource = {dblp computer science bibliography, https://dblp.org}
+}
+
+Hardt, Price, Srebro “Equality of Opportunity in Supervised Learning”, NeurIPS, 2016.
+@article{hardt2016equality,
```

Comment:
see: https://dblp.uni-trier.de/rec/conf/nips/HardtPNS16.html?view=bibtex&param=0
[with NIPS -> NeurIPS]

Reply:
I believe 2016 preceeds the renaming of the conference (?). Is it okay to call it by the new name regardless? I'm obviously very much in favor, just trying to understand the conventions.

Reply:
In this case, retroactive renaming is okay... we're providing a link, so it shouldn't matter.